### PR TITLE
Cusomizing console route DOC update

### PR DESCRIPTION
**Pull request by dpateriya**
Added note to add DNS record for custom console route

Version(s):
RHOCP 4.17

Issue:
Update the Customizing console route doc with a NOTE to add the DNS record pointing to application ingress loadbalancer.

Link to docs preview:
https://86091--ocpdocs-pr.netlify.app/openshift-enterprise/latest/web_console/customizing-the-web-console.html#customizing-the-console-route_customizing-web-console

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/customizing-the-web-console-URL.adoc [hunk 1]
```diff
@@ -53,6 +53,12 @@ spec:
 
 . Save the file to apply the changes.
 
+[NOTE]
+====
+Add a DNS record for the custom console route pointing to the application ingress load balancer.
```
Comment (lahinson) [inline modules/customizing-the-web-console-URL.adoc:58]: The current wording can be interpreted in 2 ways: 
- "For the custom console route that points to the application ingress load balancer, add a DNS record"
- "For the custom console route, add a DNS record that points to the application ingress load balancer"

Consider revising the sentence for clarity.

**Review by lahinson — COMMENTED** [review]
Hi @dpateriya! I was a bit unclear on the intended meaning of the note. When you can, PTAL at my comment and let me know what you think. Thanks!

File: modules/customizing-the-web-console-URL.adoc (hunk 1 above)
Comment (dpateriya) [reply modules/customizing-the-web-console-URL.adoc:58]: Hi @lahinson,

This looks good to me:
"For the custom console route, add a DNS record that points to the application ingress load balancer"

Comment (lahinson) [reply modules/customizing-the-web-console-URL.adoc:58]: That is clearer, thanks! I'll mark this peer review as done, assuming you'll make the update to that note.

**Review by dpateriya — COMMENTED** [review]
LGTM

File: modules/customizing-the-web-console-URL.adoc (hunk 1 above)
Comment (dpateriya) [reply modules/customizing-the-web-console-URL.adoc]: Done

**Review by dpateriya — COMMENTED** [review]
updated

**Pull request closed without merging**